src/m365/entra/commands/administrativeunit/administrativeunit-list.spec.ts

```diff
@@ -85,9 +92,8 @@ describe(commands.ADMINISTRATIVEUNIT_LIST, () => {
       throw 'Invalid request';
     });
 
-    await command.action(logger, {
-      options: {}
-    });
+    const parsedSchema = commandOptionsSchema.safeParse({});
```

Comment:
Let's use `commandOptionsSchema.parse` in these instances, since we do not check if the parse operation was succesful. Implement that for similar lines